pkg/driver/camera/camera_linux.go

```diff
@@ -158,8 +161,13 @@ func (c *camera) Open() error {
 		return err
 	}
 
-	// Late frames should be discarded. Buffering should be handled in higher level.
-	cam.SetBufferCount(1)
+	// Buffering should be handled in higher level.
+	err = cam.SetBufferCount(bufferedFrameCount)
+	if err != nil {
+		return err
+	}
+
+	c.prevFrameTime = time.Now()
```

Comment:
This will cause the first call to the readerFunction to discard a stale frame if needed. That is, a frame from more than a second ago. For streaming at >1fps this will only happen once to get the stream on track then never again.